examples/release_highlights/plot_release_highlights_1_5_0.py

```diff
@@ -55,33 +58,68 @@
 # given metric, using :class:`~model_selection.TunedThresholdClassifierCV`.
 #
 # Due to the class imbalance in this dataset, the model with the default
-# decision threshold at 0.5 has a suboptimal balanced accuracy: this classifier
-# tends to over predict the majority class.
-from sklearn.metrics import balanced_accuracy_score
+# decision threshold at 0.5 has a suboptimal balanced accuracy and F1 score:
+# this classifier tends to over predict the majority class.
+from sklearn.metrics import balanced_accuracy_score, accuracy_score, f1_score
+
+print("Default decision threshold: 0.5")
+print(f"Accuracy: {accuracy_score(y_test, clf_05.predict(X_test)):.4f}")
+print(
+    f"Balanced accuracy: {balanced_accuracy_score(y_test, clf_05.predict(X_test)):.4f}"
+)
+print(f"F1 score: {f1_score(y_test, clf_05.predict(X_test)):.4f}")
+
+# %%
+
+from sklearn.model_selection import TunedThresholdClassifierCV
+
+accuracy_clf = TunedThresholdClassifierCV(clf_05, cv=5, scoring="accuracy").fit(
+    X_train, y_train
+)
 
-print(f"Balanced accuracy: {balanced_accuracy_score(y, classifier.predict(X)):.2f}")
+print(f"Decision threshold tuned for accuracy: {accuracy_clf.best_threshold_:.4f}")
+print(f"Accuracy: {accuracy_score(y_test, accuracy_clf.predict(X_test)):.4f}")
+print(
+    f"Balanced accuracy: {balanced_accuracy_score(y_test, accuracy_clf.predict(X_test)):.4f}"
+)
+print(f"F1 score: {f1_score(y_test, accuracy_clf.predict(X_test)):.4f}")
```

Comment:
As discussed in the linked scikit-learn PR, I don't want to include this in the final PR as the results would make the message too confusing/complex.

Reply:
The fact that tuning the decision threshold for accuracy leads to near 0.5 threshold is likely a consequence that our LR model is optimizing for the [threshold independent log-loss] which is a surrogate loss for [the 0-1 loss with the 0.5 threshold].

As a result the choice of the 0.5 decision threshold and the choice of the accuracy metric are tied. But I think this is far too complex to discuss in release highlights.